product_docs/docs/biganimal/release/reference/cli/using_features.mdx

```diff
@@ -7,13 +7,13 @@ navTitle: Using BigAnimal features
 
 You can use the faraway-replica-specific CLI commands to [create](#create-a-faraway-replica), [promote](#promote-a-faraway-replica), and [get information](#get-information-on-faraway-replicas) on faraway replicas. 
 
-To update, delete, and restore faraway replicas, use the `update-cluster`, `delete-cluster`, and `restore-cluster` commands. See [Managing clusters with the CLI](/biganimal/latest/reference/cli/managing_clusters/) for more information.
+To update, delete, and restore faraway replicas, use the `cluster update`, `cluster delete`, and `cluster restore` commands. See [Managing clusters with the CLI](/biganimal/latest/reference/cli/managing_clusters/) for more information.
```

Comment:
command names needs to be updated 
faraway-replica create
faraway-replica update
faraway-replica delete

Reply:
fixed